frontend/src/scenes/events/EventsTable.tsx

```diff
@@ -562,27 +541,6 @@ export function EventsTable({
                                 >
                                     Export
                                 </LemonButtonWithPopup>
-                            ) : (
```

Comment:
fly-by: this is incorrect. We're showing the old version of the export interaction if showing exporting is turned off. 

This went wrong when the feature flag was removed